README.md

```diff
@@ -4,18 +4,17 @@ LearnHub.mk is an innovative online learning platform dedicated to nurturing the
 
 Our mission is to provide a collaborative space where individuals can enhance their web development expertise, work on real projects, and engage with a supportive community. Whether you are embarking on your coding journey or a seasoned developer, LearnHub.mk has something valuable to offer.
 
-
 ## Features
 
 ### 📚 Comprehensive Learning Resources
 
-Access a diverse range of educational materials covering various web development topics. 
+Access a diverse range of educational materials covering various web development topics.
```

Comment:
Updating the Readme is fine, but the changes in this Pull Request have nothing to do with Prettier.
Leave it like this, but for future reference -  make smaller commits based on functionality.

Reply:
https://stackoverflow.com/questions/75587906/why-airbnb-style-guide-force-eslint-rule-react-function-component-definition